scripts/extract-new-texts.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+/* eslint-disable  */
+const path = require("path");
+const program = require("commander");
+const crc32 = require("crc-32").str;
+const fs = require("fs");
+const glob = require("glob");
+const { exec } = require("child_process");
+const data = fs.readFileSync("./../src/translations/ach.json");
+const langs = JSON.parse(data);
+
+exec("git rev-parse --abbrev-ref HEAD", (err, stdout, stderr) => {
+  if (err) {
+    // handle your error
+  }
+
+  if (typeof stdout === "string") {
+    const branch_name = stdout.trim()
+
+    /** *********************************************
+     * Common
+     */
+    const globs = ["**/*.js", "**/*.ts", "**/*.tsx"];
+
+    const getKeyHash = (string) => crc32(string);
+
+    /** **********************************************
+     * Compile
+     */
+    extractTranslations();
+
+    function extractTranslations() {
+      (async () => {
+        try {
+          const file_paths = [];
+          const messages = [];
+          /*
+            (['"])                    = the capturing group of either single quotes or double quotes, at the end must be following of \_your_capturing_group_no
+            (.*?)                     = is capturing group match all characters, unlimited characters
+            \_your_capturing_group_no = is matching capturing group, for e.g. `\1` means that it will match exactly the first capturing group
+            \s*                       = it matches any whitespace characters until unlimited times in our case it used to catch `\n` or newline
+            /gs                       = `g` means it will take all of the matches, `s` means it matches even in newline
+            */
+          const i18n_marker = new RegExp(
+            /translate_text=(['"])(.*?)\1|localize\(\s*?(['"])\s*(.*?)\s*\3/gs
+          );
+         
+
+          // Find all file types listed in `globs`
+          for (let i = 0; i < globs.length; i++) {
+            let filesFound = glob.sync(`../src/${globs[i]}`);
+            filesFound = filesFound.filter(
+              (path) => path.indexOf("__tests__") === -1
+            );
+            file_paths.push(...filesFound);
+          }
+          // Iterate over files and extract all strings from the i18n marker
+          for (let i = 0; i < file_paths.length; i++) {
+            if (program.verbose) {
```

Comment:
We don't need this, right?

Reply:
This is needed. We need to get all the text to be translated this is a copy of extractor we have but slightly modified to achieved our requirements